### PR TITLE
ADD unittest && SORT package.json

### PR DESCRIPTION
**Pull request by ndelangen**
As promised, here's a PR adding a unittest to this example repo. Adding this helped me get quick feedback whether my macro was working correctly, so I think it's valuable for someone writing their first macro. Which is exactly what this repo is about.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const babel = require('babel-core');
```
Comment (kentcdodds) [inline test.js]: The recommended testing approach is to use [`babel-plugin-tester`](https://github.com/babel-utils/babel-plugin-tester). Would you mind trying that?

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Looks great! Thanks!

Could you add `yarn.lock` to the `.gitignore`? I'm not a big fan of including lockfiles in repos like this. Thanks!

**Review by kentcdodds — APPROVED** [review]
Excellent! Thanks!

**Comment by ndelangen** [comment]
🎉

**Pull request merged**